atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CQLKeyValueServiceSerializableTransactionIntegrationTest.java

```diff
@@ -24,6 +25,9 @@
 public class CQLKeyValueServiceSerializableTransactionIntegrationTest extends
         AbstractSerializableTransactionTest {
 
+    @ClassRule
+    public static CassandraResources cassandraResources = CassandraResources.getCassandraResource();
```

Comment:
A little confused about naming, especially around `CassandraResource` vs `CassandraResources`. 

* I wonder if *Resource and *Resources should be two separate classes (plurals usually denote utility classes, see eg TransactionManagers). 
* Not a fan of "Resource" in the name. Would (for example) `CassandraInstance` or `CassandraContainer` work better?